src/styles/index.scss

```
@@ -1,3 +1,144 @@
+@import './utils/variables';
+
 body {
   margin: 0;
 }
+
+img {
+  display: flex;
+  padding: 32px 19px 40px;
+  width: 160px;
+  height: 134px;
+  }
```

Use a classes selector for image
```suggestion
img {
  display: flex;
  padding: 32px 19px 40px;
  width: 160px;
  height: 134px;
}
```